pkg/cnservice/distributed_tae.go

```diff
@@ -43,9 +44,11 @@ func (s *service) initDistributedTAE(
 	}
 
 	m := mheap.New(guest.New(pu.SV.GuestMmuLimitation, pu.HostMmu))
+	proc := process.New(ctx, m, pu.TxnClient, nil, pu.FileService)
```

Comment:
Should call pu.TxnClient.New to get a TxnOperator and pass to process.New

Reply:
get it

Reply:
And here https://github.com/matrixorigin/matrixone/pull/5317/files#diff-9f173a6c762766883dd2cf2903892ee23223381f92bb6aaa1ca64b9e3702dbffR770 , should pass the process as argument

Reply:
And in partitonBatch, the process should be passed as an argument.